ci/pkl-pipelines/common/pipeline_self_update.pkl

```diff
@@ -0,0 +1,60 @@
+import "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.1#/Pipeline.pkl"
```

Comment:
Using the full import path makes the intellij plugin work, currently it can't understand the friendly names used in the PklProject file